parsons/box/box.py

```diff
@@ -0,0 +1,200 @@
+"""Class for Box API.
+
+To get authentication info (this eventually belongs in the docs for
+this connector):
+
+Information on Box site here:
+https://developer.box.com/guides/applications/custom-apps/oauth2-setup/
+
+1. Set up Box account
+2. Go to Developer console (https://app.box.com/developers/console)
+3. Either select an existing app or create a new one. To create a new one:
+  - Select "New App" > "Custom App" > "OAuth 2.0 with JWT"
+4. View your app and select "Configuration" in the left menu
+5. Scroll down to get the client id & secret, and just above it
+   select OAuth2.0 with JWT (Server Authentication) and
+   generate a developer token, aka "access token".
+
+"""
+
+import logging
+
+import boxsdk
+
+from parsons.etl.table import Table
+from parsons.utilities.check_env import check as check_env
+from parsons.utilities.files import create_temp_file
+
+import tempfile
+
+logger = logging.getLogger(__name__)
+
+
+class Box(object):
+    """Box is a file storage provider.
```

Comment:
In the docstring for some functions, we have a space after the `:` , such as
> client_id: str
and some don't have the space
> client_id:str

Also, some of the argument descriptions end in a `.` and some don't.

Would be good to have these inconsistencies addressed.

Reply:
Good point - I got sloppy.